components/ActionBar/ActionBar.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { ButtonAnchor } from '../global/Button/Button'
+import { PageMetadata } from '../global/withPageMetadata'
+import getConferenceDates from '../../config/dates'
+import dateTimeProvider from '../utils/dateTimeProvider'
+import { StyledActionBarContainer } from './ActionBar.styled'
+
+interface ActionBarProps {
+  metadata: PageMetadata
+}
+
+export const ActionBar: React.FC<ActionBarProps> = ({ metadata }) => {
+  const { AcceptingPresentations } = getConferenceDates(metadata.conference, dateTimeProvider.now())
+
+  return (
+    <StyledActionBarContainer>
+      <div>
+        <p>DDD Perth is an inclusive conference for the Perth tech community.</p>
+        {AcceptingPresentations && (
```

Comment:
This duplicates existing login in https://github.com/dddwa/dddperth-website/blob/master/config/actions.ts. Instead, we should pass in the current actions per this in `index.tsx`: `const actions = getConferenceActions(conference, dates)`.

Reply:
The actions are ordered in a particular way.

`actions[0]` will be the "primary" action at any particular time as per https://github.com/dddwa/dddperth-website/blob/master/pages/index.tsx#L95

Reply:
also, previously we had colour coding throughout the site for the different aspects e.g. voting is pink (or whatever it is). Unsure if that makes sense to retain in the new design.

Reply:
That's a good point with the actions, I'll take a look at that.

I know what you mean with the colours, I wasn't sure with the new design if that was staying or not. I'll raise in the redesign channel as it would be good to have direction on this.

Reply:
@robdmoore have updated this to use the actions util.